### DISCUSSION TITLE
Welcome to Gato-X Discussions!

### DISCUSSION THREAD

**Discussion by AdnaneKhan** [AdnaneKhan/Gato-X#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ankushgoel27** [comment]
Hi AdnaneKhan,

I am Ankush Goel and I am a full-time bug bounty hunter. I have been hunting for 4 years now and have been in the cybersecurity industry for over a decade.

I love your gato-x tool and looking forward to improving it and learning from this community regarding the issues related to GitHub actions secrutiy.
